examples/nlp/language_modeling/conf/transformer_lm_config.yaml

```diff
@@ -6,13 +6,19 @@ model:
   preproc_out_dir: null # path to store data preprocessing outputs
   
   train_ds:
-    use_tarred_dataset: true
-    tar_files: ??? # path to tarred files
-    metadata_file: ??? # metadata for tarred dataset
-    shard_strategy: scatter
+    file_name: ??? # path to file with training data
     tokens_in_batch: 4096
-    tar_shuffle_n: 256
-    
+    clean: true
```

Comment:
How about setting the config in this example for a model of size 100M or even 200M which people can use as a starting point?

Reply:
What does clean do?